c2corg_ui/static/partials/feed.html

```diff
@@ -1,76 +1,269 @@
+              <div class="bubble" uib-tooltip="{{'elevation_max'| translate}}" tooltip-append-to-body="true">
+                <span class="value-title glyphicon glyphicon-sort-by-attributes rotate-180"></span>
+              </div>
+            </div>
+            <div class="text-bubble" ng-if="::doc['document']['elevation_max']">
+              {{::doc['document']['elevation_max']}} m
+            </div>
+            <div class="parent-bubble" ng-if="::doc['document']['height_diff_up']">
+              <div class="bubble" uib-tooltip="{{'height_diff_up'| translate}}">
+                <span class="icon-height_diff"></span>
+              </div>
+            </div>
+            <div class="text-bubble" ng-if="::doc['document']['height_diff_up']">
+              {{::doc['document']['height_diff_up']}} m
+            </div>
+            <div class="parent-bubble" ng-if="::doc['document']['height_diff_difficulties']">
+              <div class="bubble" uib-tooltip="{{'height_diff_difficulties'| translate}}" tooltip-append-to-body="true">
+                <span class="value-title glyphicon glyphicon-resize-vertical"></span>
+              </div>
+            </div>
+            <div class="text-bubble" ng-if="::doc['document']['height_diff_difficulties']">
+              {{::doc['document']['height_diff_difficulties']}} m
+            </div>
+            <div class="grow">
+            </div>
+          </div>
+        </div>
+        <div class="parent-infop parent-infop-first" ng-if="::doc['document']['areas'] != null">
+          <a ng-href="{{::feedCtrl.createURLArea(doc['document']['areas'])}}" title="{{::feedCtrl.showArea(doc['document']['areas'])}}" ng-click="$event.stopPropagation()">
+            <div class="infop-bar">
+              <div class="parent-bubble">
+                <div class="bubble">
+                  <span class="glyphicon glyphicon-map-marker"></span>
+                </div>
+              </div>
+              <div class="text-bubble">
+                {{::feedCtrl.showArea(doc['document']['areas'])}}
+              </div>
+              <div class="grow">
+              </div>
+            </div></a>
+        </div>
+        <div class="parent-infop">
+          <div class="infop-bar">
+
+
+            <div class="parent-bubble" ng-repeat="activity in ::doc['document']['activities']">
+              <a href="#" class="shcom" uib-tooltip="{{activity | translate}}"> 
+                <div class="bubble">
+                  <span class="route-activity icon-{{::activity}}"></span>
+                </div>
+              </a>
+            </div>
+
+            <div class="grow">
+            </div>
+
+
+            <div class="parent-bubble">
+              <a href="#" class="link share-facebook" ng-if="::doc['document']['condition_rating']" uib-tooltip="{{'condition_rating' | translate}} : {{doc['document']['condition_rating'] | translate}}" tooltip-append-to-body="true">
```

Comment:
Just curious: what's the use of the ``share-facebook`` class? Why do you need to use a ``<a>`` here and for other data around?